src/components/AboutPage/components/AboutDesription/AboutDescription.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './AboutDescription.scss';
+
+export const AboutDesription = () => {
+  return (
+    <>
```

Comment:
I think better to change <> on <section>

Reply:
Yes, i've removed it in the next commit

Reply:
`<div>` is Ok, but for semantic markup, I think `<section>` will be better